tests/testthat/tests-clone_instat_calculation.R

```diff
@@ -1,27 +1,5 @@
-library(testthat)
-library(R6)
 library(instatCalculations)
 
-# Cloner class that includes the clone_instat_calculation method
-Cloner <- R6Class("Cloner",
-                  public = list(
-                    clone_instat_calculation = function(curr_instat_calculation, ...) {
-                      new_manips <- lapply(curr_instat_calculation$manipulations, function(x) self$clone_instat_calculation(x))
-                      new_subs <- lapply(curr_instat_calculation$sub_calculations, function(x) self$clone_instat_calculation(x))
-                      instat_calculation$new(
-                        function_exp = curr_instat_calculation$function_exp,
-                        type = curr_instat_calculation$type,
-                        name = curr_instat_calculation$name,
-                        result_name = curr_instat_calculation$result_name,
-                        manipulations = new_manips,
-                        sub_calculations = new_subs,
-                        calculated_from = curr_instat_calculation$calculated_from,
-                        save = curr_instat_calculation$save
-                      )
-                    }
-                  )
-)
-
```

Comment:
So, in our testing, if you're ever creating an R6 class then that means we're testing the wrong way. We want to remove this.